x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_actions/rule_actions_pli_essentials.cy.ts

```diff
@@ -68,6 +69,7 @@ describe(
       cy.get(WEBHOOK_ACTION_BTN).should('not.exist');
       cy.get(XMATTERS_ACTION_BTN).should('not.exist');
       cy.get(SERVER_LOG_ACTION_BTN).should('not.exist');
+      cy.get(CASES_SYSTEM_ACTION_BTN).should('not.exist');
```

Comment:
why it does not exist?
Does .cases action available only for Complete tier and Platinum license? 
If so, can you please put into PR description?

It also means, adding assertion here https://github.com/elastic/kibana/blob/8.15/x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_actions/rule_actions_pli_complete.cy.ts, that .cases displayed would be useful

Reply:
Ah yes I will add that to the PR description. The cases connector is only available with platinum license: https://github.com/elastic/kibana/blob/aa67c800ce74539caceb48c907dc34a592cd8668/x-pack/plugins/cases/server/connectors/cases/index.ts#L60

I will also add an assertion in the `complete` tier cypress test too 👍

Reply:
added here: 19748e3a45fe5f602a73b35041493244dd274718